packages/store/src/dispatcher.ts

```diff
@@ -43,7 +44,7 @@ export class InternalDispatcher {
       error: error => this._errorHandler.handleError(error)
```

Comment:
Should the error handler also be run in the angular zone? What is a reasonable expectation here?